src/sql/src/statement.rs

```diff
@@ -1377,6 +1361,31 @@ fn handle_create_source(scx: &StatementContext, stmt: Statement) -> Result<Plan,
     }
 }
 
+fn maybe_rename_columns(
```

Comment:
It would be great to have a comment here to clear up the mysterious name.

Reply:
Done.